client/src/api/eventstream.tsx

```diff
@@ -30,8 +30,8 @@ import { updateTgo } from "./tgosSlice";
 import { threatZonesUpdated } from "./threatZonesSlice";
 import { unculledZonesUpdated } from "./unculledZonesSlice";
 import { LatLng } from "leaflet";
-import { updateIadsConnection } from "./iadsNetworkSlice";
-import { IadsConnection } from "./_liberationApi";
+//import { updateIadsConnection } from "./iadsNetworkSlice";
+//import { IadsConnection } from "./_liberationApi";
```

Comment:
This is to prevent warnings from TSC because I delete stuff for IADS since it's coming back with its own PR...